docs/source/azure/policy/resources/networkinterface.rst

```diff
@@ -42,4 +48,28 @@ and then perform the delete operation on those ready for deletion.
           - type: marked-for-op
             op: delete
         actions:
-          - type: delete
\ No newline at end of file
+          - type: delete
+
+.. code-block:: yaml
+
+    policies:
+      - name: get-nic-with-user-routes
+        resource: azure.networkinterface
+        filters:
+          - type: effective-route-table
+            key: routes.value[].source
+            op: in
+            value_type: swap
+            value: User
+
+.. code-block:: yaml
+
+    policies:
+      - name: virtual-network-gateway-hop
+        resource: azure.networkinterface
+        filters:
+          - type: effective-route-table
+            key: routes.value[].nextHopType
+            op: in
+            value_type: swap
```

Comment:
fwiw, if you want to have samples without SWAP this is a perfectly good candidate as you have a list of potential values, any of which can be acceptable

`'VirtualNetworkGateway', 'VnetLocal', 'Internet', 'VirtualAppliance', 'None'`

So people might be looking for 2 of the 5 or something.  I think that is how it works if you use an array and no swap?